src/modules/shared/components/Timepicker/Timepicker.tsx

```diff
@@ -0,0 +1,9 @@
+import { Timepicker as Base, TimepickerProps } from '@meemoo/react-components';
+import { FC } from 'react';
+
+// This component only wraps in the styling
+import 'react-datepicker/dist/react-datepicker.min.css';
```

Comment:
Wat gebeurt er als zowel Date -en TimePicker in de pagina aanwezig zijn? 
Gaat die deze 2x toevoegen of gaat Next daar slim mee om tijdens de build?

Reply:
Van wat ik zag tijdens dev gaat dit blijkbaar in een global css bestand terechtkomen, bij het importeren in de `Datepicker` was de styling voor `Timepicker` ook aanwezig.